packager/react-packager/src/Resolver/polyfills/Array.prototype.es6.js

```diff
@@ -5,54 +5,51 @@
  * @polyfill
```

Comment:
@ide means www for "internal package manager". This is where @polyfill is being read. I'd love to avoid to fork them qs much as possible